Earthfile

```
@@ -1,20 +1,25 @@
+VERSION 0.6
+
 all:
     BUILD +all-test
     BUILD +all-integration-test
     BUILD +npm
 
 all-test:
-    BUILD --build-arg ELIXIR=1.9.4 --build-arg OTP=21.3.8.24 +test
-    BUILD --build-arg ELIXIR=1.12.1 --build-arg OTP=24.0.2 --build-arg RUN_INSTALLER_TESTS=1 +test
+    BUILD --build-arg ELIXIR=1.10.0 --build-arg OTP=21.3.8.24 +test
+    BUILD --build-arg ELIXIR=1.13.4 --build-arg OTP=24.3.4 --build-arg ALPINE=3.15.3 --build-arg RUN_INSTALLER_TESTS=1 +test
 
 test:
     FROM +test-setup
+    COPY --dir installer ./
     RUN MIX_ENV=test mix deps.compile
-    COPY --dir assets config installer lib integration_test priv test ./
+    COPY --dir assets config lib integration_test priv test ./
 
     # Run unit tests
     RUN mix test
 
+    ARG RUN_INSTALLER_TESTS
```

`IF [ "$RUN_INSTALLER_TESTS" = "1" ]` is failing silently, that's a fix